netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -289,7 +290,8 @@ public void handlerAdded(final ChannelHandlerContext ctx) throws Exception {
 
     // init max connection age monitor
     if (maxConnectionAgeInNanos != MAX_CONNECTION_AGE_NANOS_DISABLED) {
-      maxConnectionAgeMonitor = ctx.executor().schedule(
```

Comment:
Without this change, the `.schedule()` method is compiled as `EventLoopGroup.schedule()`. 
With this change,  the `.schedule()` method is compiled as  `ScheduleExecutorService.schedule()`.


In the runtime, if `ctx.executor()`'s value is a mockito-stubbed eventLoop, `eventLoop.schedule()` will again invokes the compiled method reflectively: `method.invoke(caller, arguments)`, where the `caller` can be the `fakeClock`. Then without this change:  it will throw `IllegalArgumentException` because the first argument `fakeClock`, is not a `EventLoopGroup` instance. With this change: `method.invoke(caller, argument)` won't fail in the runtime. 

If `ctx.executor()` is not a mockito-stubbed eventLoop, then the runtime behaviors are the same for the previous code and the changed code, because in both case it will call the `.schedule()` method of the caller's runtime type.